R/PLNfit-class.R

```diff
@@ -205,13 +209,17 @@ PLNfit <- R6Class(
     },
 
     variance_jackknife = function(Y, X, O, w, config = config_default_nlopt) {
+      cat("\n  Computing jackknife variance estimator", sep = "\n")
```

Comment:
Could you add a condition to display the message (based on the config argument) ??

Reply:
Argh, this a leftover from my debugging session. I'm inclined to remove it completely. Do you think we should keep it (if so, I'll add the same for bootstrap) ?

Reply:
removing is perfectly fine.